### PR TITLE
Add WindowExecJS method

### PR DESCRIPTION
**Pull request by leaanthony**
Exposes the existing `ExecJS` method as `WindowExecJS`

### CODE REVIEW

File: v2/pkg/runtime/window.go [hunk 1]
```diff
@@ -163,6 +163,12 @@ func WindowIsNormal(ctx context.Context) bool {
 	return appFrontend.WindowIsNormal()
 }
 
+// WindowExecJS executes the given Js in the window
+func WindowExecJS(ctx context.Context, js string) {
```
Comment (stffabi) [inline v2/pkg/runtime/window.go:167]: We are missing the implementation in the DevServer frontend. We could maybe embed the base Frontend into the DevServer so we don't need to always add a wrapper method there.

I have commit for that ready, if you want I can add it to the PR?

File: website/docs/reference/runtime/window.mdx [hunk 2]
```diff
@@ -41,6 +41,12 @@ Centers the window on the monitor the window is currently on.
 Go: `WindowCenter(ctx context.Context)`<br/>
 JS: `WindowCenter()`
 
+### WindowExecJS
+
+Executes arbitrary JS code in the window.
```
Comment (stffabi) [inline website/docs/reference/runtime/window.mdx:46]: Should we explicitly mention this is fire and forget, so incorrect/failing script snippets won't be logged out somewhere and won't panic?

File: v2/pkg/runtime/window.go (hunk 1 above)
Comment (leaanthony) [reply v2/pkg/runtime/window.go:167]: That would be great 👍 Thank you 👍

File: website/docs/reference/runtime/window.mdx (hunk 2 above)
Comment (leaanthony) [reply website/docs/reference/runtime/window.mdx:46]: Yes, great point. I'll update the docs.

File: v2/pkg/runtime/window.go (hunk 1 above)
Comment (stffabi) [reply v2/pkg/runtime/window.go:167]: Just pushed it.

File: website/docs/reference/runtime/window.mdx (hunk 2 above)
Comment (leaanthony) [reply website/docs/reference/runtime/window.mdx:46]: @stffabi - docs updated. Looks like there's a merge conflict caused by #1931 (clashes with the embed commits). Happy to look at it later but if it's obvious to you and a really quick fix then that would be cool 🙏 I think we're almost ready for v2.0.1 🎉

Comment (stffabi) [reply website/docs/reference/runtime/window.mdx:46]: Just pushed the fix for the conflicts.

Comment (stffabi) [reply website/docs/reference/runtime/window.mdx:46]: Docs looking great 👍

**Review by stffabi — APPROVED** [review]
LGTM 👍

**Pull request merged**